system-tests/e2e/case_supporting_evidence.cy.js

```diff
@@ -150,4 +150,60 @@ describe('case_supporting_evidence.cy.js', () => {
+    cy.get('#Supporting_Evidence').should('exist');
+    cy.get('.attachment-list').should('have.length', 0);
+    cy.get('[data-test="add-button"]').click();
+    cy.get('[data-test="button-input-dialog-upload-file"]').click();
+    cy.get('.drop-file-box-content').selectFile('../backend/tests/fixtures/pedigree-fake.jpg', {
+      action: 'drag-drop',
+    });
+    cy.get('.comments').type('this is a test comment for a test file');
+    cy.get('[data-test="confirm"]').click();
+    cy.get('[href="#Supporting_Evidence"]').click();
+    cy.get('.attachment-list').should('have.length', 1);
+    cy.get('.attachment-name > div').should('have.attr', 'target', '_blank');
+    cy.get('.attachment-name > div').should('have.attr', 'rel', 'noreferrer noopener');
+    cy.get('[data-test="add-button"]').click();
+    cy.get('[data-test="button-input-dialog-upload-file"]').click();
+    cy.get('.drop-file-box-content').selectFile('../backend/tests/fixtures/pedigree-fake.jpg', {
+      action: 'drag-drop',
+    });
+    cy.get('.comments').type('this is a test comment for a test file');
+    cy.get('[data-test="confirm"]').click();
+    cy.get('.attachment-list').should('have.length', 1);
+  });
+
+  it('should be able to attach the same supporting evidence file to different analyses', () => {
+    cy.get('#Supporting_Evidence').should('exist');
+    cy.get('.attachment-list').should('have.length', 0);
+    cy.get('[data-test="add-button"]').click();
+    cy.get('[data-test="button-input-dialog-upload-file"]').click();
+    cy.get('.drop-file-box-content').selectFile('../backend/tests/fixtures/pedigree-fake.jpg', {
+      action: 'drag-drop',
+    });
+    cy.get('.comments').type('this is a test comment for a test file');
+    cy.get('[data-test="confirm"]').click();
+    cy.get('[href="#Supporting_Evidence"]').click();
+    cy.get('.attachment-list').should('have.length', 1);
+    cy.get('.attachment-name > div').should('have.attr', 'target', '_blank');
+    cy.get('.attachment-name > div').should('have.attr', 'rel', 'noreferrer noopener');
+
+    cy.get('.rosalution-logo').click();
+    cy.get('[href="/rosalution/analysis/CPAM0046"] > .analysis-card > .analysis-base').click();
+    cy.get('[href="#Supporting_Evidence"]').click();
+    cy.get('.attachment-list').should('have.length', 0);
+    cy.get('[data-test="add-button"]').click();
+    cy.get('[data-test="button-input-dialog-upload-file"]').click();
+    cy.get('.drop-file-box-content').selectFile('../backend/tests/fixtures/pedigree-fake.jpg', {
+      action: 'drag-drop',
+    });
+    cy.get('.comments').type('this is a test comment for a test file');
+    cy.get('[data-test="confirm"]').click();
+    cy.get('[href="#Supporting_Evidence"]').click();
+    cy.get('.attachment-list').should('have.length', 1);
+    cy.get('.attachment-name > div').should('have.attr', 'target', '_blank');
+    cy.get('.attachment-name > div').should('have.attr', 'rel', 'noreferrer noopener');
```

Comment:
Additional verifying that the new supporting evidence being attached has the correct comment text would also ensure that it is working.